aws/resource_aws_cloudwatch_log_metric_filter.go

```diff
@@ -201,3 +207,16 @@ func resourceAwsCloudWatchLogMetricFilterDelete(d *schema.ResourceData, meta int
 
 	return nil
 }
+
+func resourceAwsCloudWatchLogMetricFilterImport(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	idParts := strings.Split(d.Id(), ":")
+	if len(idParts) != 2 || idParts[0] == "" || idParts[1] == "" {
+		return nil, fmt.Errorf("Unexpected format of ID (%q), expected <log_group_name>:<name>", d.Id())
+	}
+	logGroupName := idParts[0]
+	name := idParts[1]
+	d.Set("log_group_name", logGroupName)
+	d.Set("name", name)
+	d.SetId(logGroupName + ":" + name)
```

Comment:
Same here -- the imported resource ID should match existing resource IDs

```suggestion
	d.SetId(name)
```